lib/recurly/pricing/attach.js

```diff
@@ -53,11 +53,11 @@ exports.attach = function (el) {
       pricing = pricing.currency(dom.value(elems.currency));
     }
 
-    if (target('addon') && elems.addon) {
+    if (elems.addon && target('addon')) {
       addons();
     }
 
-    if (target('coupon') && elems.coupon) {
+    if (elems.coupon && (target('coupon') || target('plan'))) {
```

Comment:
I switched these around since the `target` method calls are a bit more expensive. Should speed things up marginally for forms not using addons or coupons.